app/people.js

```diff
@@ -61,6 +86,66 @@ const userAdded = {
   }
 }
 
+// Actions
+const inviteUser = {
+  key: "inviteUser",
+  noun: "User",
+  display: {
+    label: "Invite User",
+    description: "Invites user to the current portal."
+  },
+  operation: {
+    inputFields: [
+      {
+        label: "EMail",
+        key: "email",
+        required: true
+      },
+      {
+        label: "Role",
+        key: "type",
+        required: true,
+        choices: { "2": "User", "4": "Power user", "1": "Room admin", "3": "DocSpace admin" }
```

Comment:
Please prefer the multi-line object declaration over one-line ones. You can add a eslin rule to help with it.

```js
{
  choices: {
    "2": "User",
    "4": "Power user",
    // ...
  }
}
```